Source/VisualStyle.swift

```diff
@@ -105,7 +110,7 @@ extension VisualStyle {
 public class DefaultVisualStyle: VisualStyle {
 
     private let alertStyle: AlertControllerStyle
-    init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
+    public init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
```

Comment:
init was marked as internal (by default), so before my change it wasn't available for subclasses outside the library.